coordinator.py

```diff
@@ -66,6 +66,7 @@ def _update_gamestate(self):
         # Get a new gamestate copy from the coordinator
         try:
             self.gs = self.data_in_q.get(timeout=1)

```

Comment:
I would add a comment here explaining what this does - so any calls to logger inside gamestate functions will go to the provider's logger.